src/QMCDrivers/VMC/VMCBatched.cpp

```diff
@@ -221,13 +224,20 @@ void VMCBatched::runVMCStep(int crowd_id,
   crowd.setRNGForHamiltonian(rng);
 
   int max_steps = sft.qmcdrv_input.get_max_steps();
-  // \todo delete
-  RealType cnorm = 1.0 / static_cast<RealType>(crowd.size());
   IndexType step = sft.step;
   // Are we entering the the last step of a block to recompute at?
   bool recompute_this_step = (sft.is_recomputing_block && (step + 1) == max_steps);
   advanceWalkers(sft, crowd, timers, *context_for_steps[crowd_id], recompute_this_step);
```

Comment:
I saw accumulate_this_step a very useful flag in DMC advanced walkers. It can be controlled not only for warm up but also for frequency. I mean accumulate every N steps. So could you align VMC driver with DMC driver and also move "crowd.accumulate(rng);" into the end of collectables_timer scope of VMCBatched::advanceWalkers. Also make sure warmupsteps doesn't collect data.

Reply:
Done